docs/_static/spec/rasa.yml

```
@@ -98,6 +98,10 @@ paths:
                     type: string
                     description: Path of the loaded model
                     example: 20190429-103105.tar.gz
+                  n_training_jobs:
+                    type: integer
+                    description: Number of actively running training processes
```

optional: the word `actively` doesn't add anything to the description. Hence, I'd just leave it out.
```suggestion
                    description: Number of running training processes
```